app/javascript/packages/components/tag.spec.tsx

```diff
@@ -0,0 +1,26 @@
+import { render } from '@testing-library/react';
+import Tag from './tag';
+
+describe('Tag', () => {
+  it('renders a tag element with children', () => {
+    const { getByText } = render(<Tag>Recommended</Tag>);
+
+    const tag = getByText('Recommended');
+
+    expect(tag.classList.contains('usa-tag')).to.be.true();
+  });
+
+  it('renders with additional HTML attribute props', () => {
+    const { getByText } = render(
+      <Tag id="recommended-tag" className="example">
+        Recommended
+      </Tag>,
+    );
+
+    const tag = getByText('Recommended');
+
+    expect(tag.id).to.equal('recommended-tag');
+    expect(tag.classList.contains('usa-tag')).to.be.true();
```

Comment:
do we need this expect here considering line 10?

Reply:
I see the value of also including it here is to prevent regressions for an implementation which would use the custom `className` as a _substitute_ of the default class, rather than an additional class. This test case ensures that both would be applied.